sonar-text-plugin/src/main/resources/org/sonar/plugins/secrets/configuration/zuplo.yaml

```diff
@@ -0,0 +1,27 @@
+provider:
+  metadata:
+    name: Zuplo
+    category: Development
+    message: Make sure this Zuplo API key gets revoked, changed, and removed from the code.
+
+  rules:
+    - id: zuplo-api-key
+      rspecKey: S6753
+      metadata:
+        name: Zuplo API keys should not be disclosed
+      detection:
+        pre:
+          include:
+            content:
+              - zpka_
+        matching:
+          pattern: "\\b(zpka_[a-zA-Z\\d]{32}_[a-zA-Z\\d]{8})\\b"
```

Comment:
`[a-zA-Z\\d]` can be simplified to `\w`

Reply:
There are not exactly the same. `\w` includes `_`. This is why I have not used it.

Reply:
My mistake I misread the thing